test/src/e2e_vm_tests/test_programs/should_pass/stdlib/vec/src/main.sw

```diff
@@ -1810,3 +1811,27 @@ fn test_vector_with_capacity_u64() {
         Option::Some(val) => revert(0), Option::None => (), 
     }
 }
+
+fn test_vector_swap() {
```

Comment:
I suggest that we expand the `swap` tests to cover all possible data types. For inspiration, take a look at this PR that did that recently. So, for each function `test_vector_new_*`, append some lines that test `swap` for the given data type.

Reply:
The referenced PR is #2170

Reply:
Lol thank you @Braqzen!

Reply:
I agree with @Braqzen. How about we start by creating a new test called `vec_swap` which is similar to `vec` but tests on swaps (for all data types)? We can then further split the existing `vec` test as we see fit.

Reply:
> I agree with @Braqzen. How about we start by creating a new test called `vec_swap` which is similar to `vec` but tests on swaps (for all data types)? We can then further split the existing `vec` test as we see fit.

Do you mean creating new project `vec_swap` in e2e tests something like test/src/e2e_vm_tests/test_programs/should_pass/stdlib/vec?

Reply:
Yup! Just a separate test called should_pass/stdlib/vec_swap for example.